branding/supplemental-ui/uyuni/uyuni-webui/js/vendor/langSelection.js

```diff
@@ -0,0 +1,11 @@
+function selectLanguage(langId){          
```

Comment:
```suggestion
function selectLanguage(langId) {
```
Removing trailing whitespace and adding a blank before the `{`. I would recommend to get familiar with Javascript coding style best practices in general, and even to enforce common guidelines using a linter (e.g. [jslint](https://www.jslint.com/)) as part of the CI.